tests/TestCase/Validation/ValidationTest.php

```diff
@@ -3104,4 +3099,10 @@ public function testImageWidth()
         $this->assertFalse(Validation::imageWidth($upload, '>', 2000));
         $this->assertFalse(Validation::imageWidth($upload, '==', 3000));
     }
+
+    public function testContainNonAlphaNumericAndCCWithScalar()
```

Comment:
Why scalar? Those are non scalar values - also the 2nd one is not the same as in the previous commit (it now is a scalar).